auth_lib/models.py

```diff
@@ -18,6 +19,55 @@ def generate_random_word():
     return ''.join(choices(ascii_lowercase, k=32))
 
 
+class Session:
+
+    def __init__(self, user, id=None, data="{}", updated=None):
+        self.id = id
+        self.user = user
```

Comment:
```suggestion
        self.user_email = user_email
```
for better readability